recipes/muq/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set version = "0.3.1" %}
+
+package:
+  name: muq
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/mituq/muq2/downloads/muq_{{ version | replace(".", "_") }}.tar.gz
+  sha256: 676777a09106e44c8b285f6802cbb2777c6799ee905ca82b1d631aa280f298e2
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - numpy
+    - hdf5
+    - boost-cpp ==1.74
```

This should be allowed to come from the global conda-forge pinning:
```suggestion
    - boost-cpp
```